packages/cli/src/index.ts

```diff
@@ -0,0 +1,20 @@
+import { Command } from 'commander';
+import { createPieceCommand } from './lib/commands/create-piece';
+
+const program = new Command();
+
+program.version('0.0.1').description('Activepieces CLI');
```

Comment:
Please create alias for the cli in package.json

Reply:
and please check what's needed to be updated in the documentation as well delete the current code for create piece under tools